app/Console/Command/ReplayCommand.php

```diff
@@ -121,6 +130,12 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $cdbids = $input->getOption(self::OPTION_CDBID);
 
+        if ($cdbids !== null) {
+            foreach ($cdbids as $cdbid) {
```

Comment:
This could be an issue when we have multiple cdbids, then we purge everything instead of doing it one by one before the replay starts for one item.

Reply:
I will try to explain the issue with an example:
- A user provides the cdbids 1, 2, 3, 4 and 5
- The scripts starts and purges 1 till 5
- The script will now publish the event streams
- There is an issue and the script fails on id 2
- The ids 3, 4 and 5 are already purged but not replayed
- So we are in a situation where the state is worse then before

Reply:
Indeed, after discussing this in Slack we decided that we will only purge the first item because this Command is mostly run by helpdesk via Jenkins, and in Jenkins it is limited to 1 cdbid.